bundles/org.openhab.binding.deconz/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -335,6 +335,29 @@
 		<state readOnly="true" />
 	</channel-type>
 
+	<thing-type id="firesensor">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="deconz" />
+		</supported-bridge-type-refs>
+		<label>Fire Sensor</label>
+		<description>A fire sensor</description>
+		<channels>
+			<channel typeId="fire" id="fire" />
+			<channel typeId="last_updated" id="last_updated" />
+		</channels>
+
+		<representation-property>uid</representation-property>
+
+		<config-description-ref uri="thing-type:deconz:sensor" />
+	</thing-type>
+
+	<channel-type id="fire">
+		<item-type>Switch</item-type>
+		<label>Fire</label>
+		<description>Fire was triggered.</description>
```

Comment:
```suggestion
		<description>A fire was detected.</description>
```

Reply:
I'm not able to accept that change.Can you do that?

Reply:
Yes, of course. Done.